### PR TITLE
[ES]Localization managed services / Passing Checks

### LINKED ISSUE #611: [Spanish] localize 'Managed services'
Localize the `content/en/managed_services.md` document to Spanish

### PR DESCRIPTION
**Pull request by ShankyJS**
### Describe your changes
Localization for `managed services`

### Related issue number or link (ex: `resolves #issue-number`)
Resolves #611

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/managed-services.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+## ¿Qué es?
+
+Un servicio administrado es una solución de software en la que las operaciones y gestión son manejadas por un tercero.
+Algunos ejemplos incluyen Bases de Datos como Servicio como Amazon RDS o un servicio de monitoreo externo como Datadog.
+
+## Problema que aborda
+
+La gestión del software es compleja, especialmente si consideramos todas las diferentes tecnologías que componen un stack moderno.
```
Comment (electrocucaracha) [inline content/es/managed-services.md:15]: nit
```suggestion
La gestión del software es compleja, especialmente si consideramos todas las  tecnologías diferentes que componen un stack moderno.
```

**Review by electrocucaracha — APPROVED** [review]

**Comment by ShankyJS** [comment]
> 

Sorry! Going to fix that one

**Review by electrocucaracha — APPROVED** [review]

File: content/es/.wordlist.txt [hunk 2]
```diff
@@ -116,12 +118,14 @@ md
 menu
 Mercurial
 mesh
+methodology
```
Comment (ramrodo) [inline content/es/.wordlist.txt:121]: I think this word is not being used

```suggestion
```

Comment (ramrodo) [reply content/es/.wordlist.txt:121]: NVM, that word appears in another document at:
https://github.com/cncf/glossary/blob/dev-es/content/es/agile-software-development.md?plain=1#L5

**Review by ramrodo — APPROVED** [review]
/lgtm

**Review by 92nqb — APPROVED** [review]

**Pull request merged**